<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBSDStartPage</class>
 <widget class="QWidget" name="WBSDStartPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QFrame" name="connStatusPic">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>528</width>
        <height>150</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLabel" name="connStatusText">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="connStatusInd">
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="spacing">
      <number>7</number>
     </property>
     <item row="1" column="1">
      <widget class="QPushButton" name="generalButton">
       <property name="text">
        <string>General settings</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="timeButton">
       <property name="text">
        <string>Time and events</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="displayButton">
       <property name="text">
        <string>Display settings</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="countersButton">
       <property name="text">
        <string>Counters</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="testButton">
       <property name="text">
        <string>Testing</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="waterButton">
       <property name="text">
        <string>Water system</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="drinkButton">
       <property name="text">
        <string>Drink settings</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="errorButton">
       <property name="text">
        <string>Errors</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="payButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Payment/RFID settings</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btnlan">
       <property name="text">
        <string>Clone/backup and restore</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>errorButton</tabstop>
  <tabstop>waterButton</tabstop>
  <tabstop>countersButton</tabstop>
  <tabstop>generalButton</tabstop>
  <tabstop>drinkButton</tabstop>
  <tabstop>displayButton</tabstop>
  <tabstop>timeButton</tabstop>
  <tabstop>payButton</tabstop>
  <tabstop>testButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
